server/models/Notification.js

```diff
@@ -132,6 +132,28 @@ export default function(Sequelize, DataTypes) {
       },
     }).then(us => us.map(us => us.UserId));
   };
+  /**
+   * Check if notification with `notificationType`, `CollectiveId` and `UserId` is active.
+   * @param {*} notificationType
+   * @param {*} CollectiveId
+   * @param {*} UserId
+   */
+  Notification.isActive = (notificationType, CollectiveId, UserId) => {
+    debug('isActive', notificationType, CollectiveId, UserId);
+    return models.Notification.findOne({
+      where: {
+        CollectiveId,
+        type: notificationType,
+        UserId,
+      },
+    }).then(notification => {
+      if (notification && notification.active) {
```

Comment:
If I'm not mistaken, an absence of notification entry in the table should be considered active. Right?

Reply:
No, not active.

Reply:
It should be active by default I think.